code/implementation/docker/machine/src/main/resources/schema/base/amazonec2Config.json.d/amazonec2.json

```diff
@@ -12,8 +12,7 @@
         },
         "vpcId": {
             "type": "string",
```

Comment:
yeah, I think we should add the check in the UI but for now, since it's so specific to amazon, we'll let docker-machine handle it.  I don't want to get a place where docker-machine changes it's requirement on us and we have to change cattle just to accommodate it.

This does mean we might need to think about how far we want to consider supporting these "first class" drivers.  There's so much more we can do here before sending the values down to machine.